doc/supported-platforms.md

```diff
@@ -10,6 +10,7 @@ Ignition is currently only supported for the following platforms:
 * [Packet] - Ignition will read its configuration from the instance userdata. SSH keys are handled by coreos-metadata.
 * [QEMU] - Ignition will read its configuration from the 'opt/com.coreos/config' key on the QEMU Firmware Configuration Device (available in QEMU 2.4.0 and higher).
 * [DigitalOcean] - Ignition will read its configuration from the droplet userdata. SSH keys and network configuration are handled by coreos-metadata.
+* [zVM] - Ignition will read its configuration from the reader device directly.
```

Comment:
Does the data need to be formatted in a particular way in order for it to be read by this code? Or is this a standard scheme that everyone on Z already knows?

Reply:
Similarly, as someone not familiar with zVM, I don't know what the reader device is. But if it's something people on Z would know, this is fine.

Reply:
Reader device is z/VM virtual unit record devices.There will not have gap for people familiar with zVM platform.